holmes/plugins/toolsets/kafka.py

```diff
@@ -496,11 +490,13 @@ def _invoke(self, params: Dict) -> StructuredToolResult:
                         consumer_group_description_future.result()
                     )
                     bootstrap_servers = self.get_bootstrap_servers(kafka_cluster_name)
+                    topic_metadata = client.list_topics(topic_name, timeout=10)
```

Comment:
according to the docs I see, the return type is ClusterMetadata, thus it can be used here and in the group_has_topic instead of the 'Any' type

Reply:
Good catch